weedcoco/tests/repo/deposit_data/basic_1/weedcoco.json

```diff
@@ -0,0 +1,732 @@
+{
+    "images": [
+        {
+            "id": 1,
+            "file_name": "/Users/elevn/Documents/workspace/Weed-ID-Interchange/weedcoco/tests/repo/deposit_data/basic_1/images/001_image.png",
```

Comment:
Hmm... I had expected input to have relative paths, i.e. just filenames.

I've not reviewed how you currently handle absolute paths. I think we can assume that all image files are in one directory, and just use the basename to reference them in a collection of uploaded images.